R/miic.R

```diff
@@ -92,17 +92,21 @@
 #' propagated to downstream undirected edges in unshielded triples following
 #' the orientation method
 #'
-#' @param state_order [a data frame]
-#' An optional d*(2-3) data frame giving the order of the ordinal categorical variables.
-#' It will be used during post-processing to compute the signs of the edges using partial
-#' linear correlation. 
-#' If specified, the data frame must have at least a "var_names" column, containing the
-#' names of each variable as specified by colnames(input_data). A "var_type" column may
-#' specify if each variable is to be considered as discrete (0) or continuous (1). And 
-#' the "levels_increasing_order" column contains a single character string with all of
-#' the unique levels of the ordinal variable in increasing order, delimited by a comma.
-#' If the variable is categorical but not ordinal, the "levels_increasing_order" column
-#' may instead contain NA.
+#' @param state_order [a data frame] An optional data frame providing extra
+#' information for variables. It must have d rows where d is the number of input
+#' variables, and the following structure (named columns):
+#'
+#' "var_names" (required) contains the name of each variable as specified
+#' by colnames(input_data).
+#'
+#' "var_type" (optional) contains a binary value that specifies if each
+#' variable is to be considered as discrete (0) or continuous (1).
+#'
+#' "levels_increasing_order" (optional) contains a single character string
+#' with all of the unique levels of the ordinal variable in increasing order,
+#' delimited by comma ','. It will be used during the post-processing to compute
+#' the sign of an edge using partial linear correlation. If the variable is
+#' categorical but not ordinal, this column may instead contain NA.
```

Comment:
This should also be empty or NA for continuous variables. Though it's obvious for us, maybe for users it's important to make it clear.

Reply:
See above, I suggest leave those to another separate doc-revision PR, this PR only concerns the new parameter.